src/reports/RouteReportPage.jsx

```diff
@@ -162,10 +170,26 @@ const RouteReportPage = () => {
                       />
                     </TableCell>
                   ))}
+                  {!readonly && (
+                    <TableCell style={{ paddingTop: '0px', paddingBottom: '0px' }}>
```

Comment:
Buttons have a pretty large top/bottom maring that doesn't look very good. The first icon does ` padding="none"`, but that also removes left/right margin which I would want to keep for this one in order for it to not hug the right screen edge.

Let me know if there is a better method of doing this.

Reply:
Can we reuse `CollectionActions`?

Reply:
No, I don't think so. It assumes that the model is editable, which isn't true for positions

Reply:
We can make edit optional?

Reply:
Yeah, I guess I could try that. I will give it a go later today.

I haven't tried it out yet, it seems to move actions (just one in this case) into a dropdown menu, at least on mobile. The gps icon on the left definitely stay as it is at the moment, right?

Reply:
I changed it over, let me know if it looks good to you


https://github.com/user-attachments/assets/4837e6e1-2fbb-4a6a-8f53-46dd72ba89ca